dashboard/lib/projects_list.rb

```diff
@@ -224,7 +224,8 @@ def extract_data_for_featured_project_cards(project_featured_project_user_combo_
           "type" => project_details[:project_type],
           "publishedAt" => project_details[:published_at],
           "studentName" => UserHelpers.initial(project_details[:name]),
-          "studentAgeRange" => UserHelpers.age_range_from_birthday(project_details[:birthday])
+          "studentAgeRange" => UserHelpers.age_range_from_birthday(project_details[:birthday]),
+          "isFeatured" => true
```

Comment:
Should there be a corresponding change for returning isFeatured = false somewhere?  (I see that the code will work functionally, but it might clarify intent to have isFeatured=false in the appropriate places?)